### PR TITLE
Watcher with appdynamics integration

### PR DESCRIPTION
**Pull request by mohabusama**
Sync AppDynamics job files for containers

### CODE REVIEW

**Comment by szuecs** [comment]
:+1:

**Comment by mikkeloscar** [comment]
:+1:

**Comment by mohabusama** [comment]
👍

File: k8s_log_watcher/kube.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+
+DEFAULT_SERVICE_ACC = '/var/run/secrets/kubernetes.io/serviceaccount'
+
+config = pykube.KubeConfig.from_file(DEFAULT_SERVICE_ACC)
+kube_client = pykube.HTTPClient(config)
+
+
+def get_pods() -> list:
+    return pykube.Pod.objects(kube_client).filter(namespace='dafault')
```
Comment (linki) [inline k8s_log_watcher/kube.py]: typo: `dafault`?

Comment (mohabusama) [inline k8s_log_watcher/kube.py]: True. Will fix it.

**Pull request merged**